shell/platform/linux/fl_texture.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_texture.h"
+#include "flutter/shell/platform/linux/fl_texture_private.h"
+
+#include <epoxy/gl.h>
+#include <gmodule.h>
+#include <cstdio>
+
+// Added here to stop the compiler from optimising this function away.
+G_MODULE_EXPORT GType fl_texture_get_type();
+
+G_DEFINE_TYPE(FlTexture, fl_texture, G_TYPE_OBJECT)
+
+static void fl_texture_class_init(FlTextureClass* klass) {}
+
+static void fl_texture_init(FlTexture* self) {}
+
+gboolean fl_texture_populate(FlTexture* self,
+                             uint32_t width,
+                             uint32_t height,
+                             FlutterOpenGLTexture* opengl_texture,
+                             GError** error) {
+  uint32_t target = 0, name = 0;
+  if (!FL_TEXTURE_GET_CLASS(self)->populate(self, &target, &name, &width,
+                                            &height, error)) {
+    return false;
```

Comment:
false -> FALSE (for consistency).

Reply:
I think this is resolved, I haven't found single instance in PR where `gboolean` is`false`;